### PR TITLE
fix: return the url of the final release rather than the url the draft

### LINKED ISSUE #212: Untagged GitHub Release
## Current behavior

I'm not sure what's triggering [this PR comment](https://github.com/adobe/generator-tsx/pull/59#issuecomment-522158906), but if you click on the "GitHub Release" link you'll see that it references an untagged release, but the release is [definitely tagged](https://github.com/adobe/generator-tsx/releases/tag/v1.2.6).

## Expected behavior

"GitHub Release" links to tagged release.

## Environment

```json
    "semantic-release": "^15.13.21",
```
- CI environment: Travis-CI
- Plugins used: https://github.com/jedmao/semantic-release-npm-github-config/blob/2dd093c9395a9d2a8c37564bb8d768faa7c1f378/.releaserc.json
- **semantic-release** configuration: see above
- CI logs: https://github.com/adobe/generator-tsx/runs/195735455

### PR DESCRIPTION
**Pull request by michael-wolfenden**
Fixes #212

### CODE REVIEW

**Review by gr2m — COMMENTED** [review]
<a href='https://hearts.dev/projects/3/users/gr2m'></a>

Looks good to me 👍 

Could you maybe add a test so we don't break that behavior in future?

**Comment by michael-wolfenden** [comment]
@gr2m I have updated the mock data returned by the github api to reflect the `untagged` url returned for draft releases. This causes the tests to fail for the existing code, but pass with my updates.

File: test/publish.test.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ test.serial('Publish a release with one asset', async t => {
   };
   const nextRelease = {version: '1.0.0', gitHead: '123', gitTag: 'v1.0.0', notes: 'Test release note body'};
   const options = {branch: 'master', repositoryUrl: `https://github.com/${owner}/${repo}.git`};
+  const untaggedReleaseUrl = `https://github.com/${owner}/${repo}/releases/untagged`;
```
Comment (gr2m) [inline test/publish.test.js]: just a minor, but the URL returned by github ends with `-[commit hash]`, so I would suggest we do the same here

```suggestion
  const untaggedReleaseUrl = `https://github.com/${owner}/${repo}/releases/untagged-123`;
```

And adopt the tests accordingly. Other than that this is good to go :)

File: test/publish.test.js [hunk 2]
```diff
@@ -146,6 +147,7 @@ test.serial('Publish a release with one asset and custom github url', async t =>
   };
   const nextRelease = {version: '1.0.0', gitHead: '123', gitTag: 'v1.0.0', notes: 'Test release note body'};
   const options = {branch: 'master', repositoryUrl: `https://github.com/${owner}/${repo}.git`};
+  const untaggedReleaseUrl = `${env.GH_URL}/${owner}/${repo}/releases/untagged`;
```
Comment (gr2m) [inline test/publish.test.js]:
```suggestion
  const untaggedReleaseUrl = `https://github.com/${owner}/${repo}/releases/untagged-123`;
```

File: test/publish.test.js [hunk 3]
```diff
@@ -190,6 +192,7 @@ test.serial('Publish a release with an array of missing assets', async t => {
   const pluginConfig = {assets: [emptyDirectory, {path: 'missing.txt', name: 'missing.txt'}]};
   const nextRelease = {version: '1.0.0', gitHead: '123', gitTag: 'v1.0.0', notes: 'Test release note body'};
   const options = {branch: 'master', repositoryUrl: `https://github.com/${owner}/${repo}.git`};
+  const untaggedReleaseUrl = `https://github.com/${owner}/${repo}/releases/untagged`;
```
Comment (gr2m) [inline test/publish.test.js]:
```suggestion
  const untaggedReleaseUrl = `https://github.com/${owner}/${repo}/releases/untagged-123`;
```

**Comment by michael-wolfenden** [comment]
@gr2m I have made the proposed changes

**Review by gr2m — APPROVED** [review]
<a href='https://hearts.dev/projects/3/users/gr2m'></a>

perfect thanks 👌

**Pull request merged**